src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -14,5 +17,24 @@ public FieldNameLabel(Field field) {
         setPadding(new Insets(4, 0, 0, 0));
         setAlignment(Pos.CENTER);
         setPrefHeight(Double.POSITIVE_INFINITY);
+        setTip(field);
+    }
+
+    public void setTip(Field field) {
+        Tooltip tip = new Tooltip();
+        if (StandardField.AUTHOR.getName().equals(field.getName())) {
```

Comment:
Compare field to StandardField.AUTHOR.
Forget about getName() at all. ;-)

Reply:
```suggestion
       StandardField standardField = null;
        try {
            standardField = StandardField.valueOf(field.getName().toUpperCase());
        }catch (Exception exception){
        }
        switch(standardField){
            case AUTHOR:
                tip.setText(Localization.lang("The name(s) of the author(s), in the format described in the LATEX book. Remember, all names are separated with the and keyword, and not commas."));
                break;
            case BIBTEXKEY:
                tip.setText(Localization.lang("[First author'last name][Article year] e.g. Jones2020"));
                break;
            case JOURNAL:
                tip.setText(Localization.lang("Journal name. Abbrevations could be used."));
                break;
            case TITLE:
                tip.setText(Localization.lang("The work's title"));
                break;
            case YEAR:
                tip.setText(Localization.lang("The year of publication or, for an unpublished work, the year it was written. Generally it should consist of four numerals, such as 1984, although the standard styles can handle any year whose last four nonpunctuation characters are numerals, such as '(about 1984)'."));
                break;
            default:
                return;
        }
        this.setTooltip(tip);
```

I changed code to this. I think maybe it's more concise?

Reply:
Yes that's much better!

Reply:
Instead of the try catch, you can test `field instanceof StandardField` an then cast it to, so you don't have to check for the name. As I said: forget about getName(). :-)

Reply:
Field also provides `isStandardField()`

Reply:
```suggestion
       if (field.isStandardField()) {
            standardField = (StandardField)field;
            switch (standardField) {
                case AUTHOR:
                    tip.setText(Localization.lang("The name(s) of the author(s), in the format described in the LATEX book. Remember, all names are separated with the and keyword, and not commas."));
                    break;
                case BIBTEXKEY:
                    tip.setText(Localization.lang("[First author'last name][Article year] e.g. Jones2020"));
                    break;
                case JOURNAL:
                    tip.setText(Localization.lang("Journal name. Abbrevations could be used."));
                    break;
                case TITLE:
                    tip.setText(Localization.lang("The work's title"));
                    break;
                case YEAR:
                    tip.setText(Localization.lang("The year of publication or, for an unpublished work, the year it was written. Generally it should consist of four numerals, such as 1984, although the standard styles can handle any year whose last four nonpunctuation characters are numerals, such as '(about 1984)'."));
                    break;
                default:
                    return;
            }
            this.setTooltip(tip);
```

It works well for StandardFields except "BIBTEXKEY", I think because it's not a standardfield for bibtex. Even though I add it to the enum StandardField. 
`BIBTEXKEY("bibtexkey"),`

Reply:
BibtexKey is InternalField.KEY_FIELD
You need to handle it separate.

Reply:
Got it. I'll try

Reply:
```suggestion
        if (field.isStandardField()) {
            StandardField standardField = (StandardField)field;
            switch (standardField) {
                case AUTHOR:
                    tip.setText(Localization.lang("The name(s) of the author(s), in the format described in the LATEX book. Remember, all names are separated with the and keyword, and not commas."));
                    break;
                case JOURNAL:
                    tip.setText(Localization.lang("Journal name. Abbrevations could be used."));
                    break;
                case TITLE:
                    tip.setText(Localization.lang("The work's title"));
                    break;
                case YEAR:
                    tip.setText(Localization.lang("The year of publication or, for an unpublished work, the year it was written. Generally it should consist of four numerals, such as 1984, although the standard styles can handle any year whose last four nonpunctuation characters are numerals, such as '(about 1984)'."));
                    break;
                default:
                    return;
            }
        }
        else if (field instanceof InternalField) {
            InternalField InternalField = (InternalField) field;
            switch (InternalField) {
                case KEY_FIELD:
                    tip.setText(Localization.lang("[First author'last name][Article year] e.g. Jones2020"));
                    break;
                default:
                    return;
            }
        }
        this.setTooltip(tip);
```
Yes, it works. But I'm not quite sure about the context for each tip. I want it to be short and concise, but if I set the context using comments out of the official documentation of biblatex, it will be a long paragraph. Do you have any idea?

Reply:
Please: variables lowercase

Reply:
> Please: variables lowercase

My fault. I have changed it.